packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -244,7 +244,8 @@ export const getStandardOptionEditors = () => {
     name: 'Radio',
     description: 'Allows option selection',
     editor(props) {
-      return <RadioButtonGroup {...props} options={props.item.settings?.options} />;
+      const value = props.value === undefined ? props.item.defaultValue : props.value;
+      return <RadioButtonGroup {...props} value={value} options={props.item.settings?.options} />;
```

Comment:
this feels wrong (to have the default be applied manually in every editor), hm...

Reply:
Agree, I think the default value should be applied on a higher level.  I think it should be applied in the builder API implementation as there we have access to both std editor item and option config: https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/utils/OptionsUIBuilders.ts, when `addCustomEditor` is called. 

Think the decision that we need to make is about what is the default if the user hasn't configured the `defaultValue` when using the API.

- for number: `undefined`
- for slider: we can either stick with `undefined` as default or use `min` from settings if provided
- for boolean: `undefined` or `false`
- for string: `undefined` or empty string
- for Select: `undefined` 
- for RadioButtonGroup: think we need to default to 1st item from the provided options list
- for Unit and Color: `undefined`